source/includes/tutorial-setup.rst

```diff
@@ -0,0 +1,151 @@
+.. _kafka-tutorials-setup:
+
+Before You Get Started
+----------------------
+
+The tutorials in this section use a Docker development environment to
+package the dependencies and configurations you need to run the
+{+connector-long+}. Make sure you complete the development environment setup
+steps before proceeding to the tutorials.
+
+Requirements
+~~~~~~~~~~~~
+
+To download the Docker containers you need for this tutorial, you must have
+a Docker account. To learn how to sign up for an account and install Docker
+Desktop, read the `Docker Hub Quickstart <https://docs.docker.com/docker-hub/>`__.
+
+While not required, we recommend that you use **git** to download the setup
+files. To learn how to install git, read the `Git Downloads <https://git-scm.com/downloads>`__
+page.
```

Comment:
Suggestion: 
1. For pre-reqs, I think its better to bullet/number the items
2. I think these items can be a bit more simplified since it should only be stating things they need/might need

```suggestion
- Create a Docker account and download the Docker containers you need for 
this tutorial. To learn how to sign up for an account and install Docker
Desktop, read the `Docker Hub Quickstart <https://docs.docker.com/docker-hub/>`__.

- (Optional) Install **git** to download the setup
files. To learn how to install git, read the `Git Downloads <https://git-scm.com/downloads>`__
page.
```